python/py-nbxmpp/Portfile

```diff
@@ -1,35 +1,39 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           python 1.0
+PortSystem           1.0
+PortGroup            python 1.0
+PortGroup            gitlab 1.0
 
-name                py-nbxmpp
-version             2.0.6
-revision            0
-categories-append   net
-platforms           darwin
-license             GPL-3
-supported_archs     noarch
+gitlab.instance      https://dev.gajim.org
+gitlab.setup         gajim python-nbxmpp 3.0.1 nbxmpp-
 
-python.versions     27 37 38 39 310
+name                 py-nbxmpp
+version              3.0.1
+revision             0
```

Comment:
presumably `name` and `version` are set automatically from the `gitlab.setup` line?

Reply:
> presumably `name` and `version` are set automatically from the `gitlab.setup` line?

You are right, thanks. Removed.

Reply:
sorry my bad... the "name" is indeed set from the "gitlab.setup" line but now isn't the same as the directory name. So you were right that you do need the explicit "name" field here.